src/DSharp.Compiler.Tests/Source/Expression/Generics2/Baseline.txt

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+define('g_b4bdbc8e390b4f5c8e02af2037e648c6', ['ss'], function(ss) {
```

Comment:
You'll want a ScriptAssemlyAttribute here to stop it from using a generated name. Otherwise it'll fail between runs - Something I want to fix!